extconfig/extconfig.go

```diff
@@ -19,6 +19,7 @@ type Specification struct {
 	AdviceSingleReplicaMinReplicas         int      `json:"adviceSingleReplicaMinReplicas" split_words:"true" required:"false" default:"2"`
 	DisableDiscoveryExcludes               bool     `required:"false" split_words:"true" default:"false"`
 	LogKubernetesHttpRequests              bool     `required:"false" split_words:"true" default:"false"`
+	DiscoveryDisabledArgoRollout           bool     `json:"discoveryDisabledArgoRollout" required:"false" split_words:"true" default:"true"`
```

Comment:
We usually default to "false" here and do the real defaulting via the helm values. At least your current impl of the helm chart wouldn't ever set this flag to "false"